include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -187,6 +187,7 @@ def TT_StoreOp : TT_Op<"store",
 //
 def TT_AtomicRMWOp : TT_Op<"atomic_rmw", [SameOperandsAndResultShape,
                                           SameOperandsAndResultEncoding,
+                                          MemoryEffects<[MemRead]>,
```

Comment:
I do think the rmw op also reads memory.

Reply:
Yeah my comment doesn't apply to global mem loads since it's exposed on Triton